test/controllers/tasks_controller_test.rb

```diff
@@ -79,48 +82,122 @@
 
       new_task = Task.find_by(name: task_hash[:task][:name])
       expect(new_task.description).must_equal task_hash[:task][:description]
-      expect(new_task.due_date.to_time.to_i).must_equal task_hash[:task][:due_date].to_i
-      expect(new_task.completed).must_equal task_hash[:task][:completed]
+      expect(new_task.completed).must_equal nil
+      #Need to change to 'assert_nil' but having trouble finding correct syntax and documentation for this
 
       must_respond_with :redirect
       must_redirect_to task_path(new_task.id)
     end
+
+    it "will raise an error for creating a task with invalid arguments" do 
+      expect { post tasks_path, params: invalid_task_hash }.must_differ "Task.count", 0
+      must_respond_with 200
+      #Weird bug here I think? I want to figure out how to check for a rendered form, but my page redirects to /tasks despite displaying the form again with if invalid entry. On StackOverflow I found out it might be because #index is before #create in routes - but I also can't figure out how to rewrite the order of that.
```

Comment:
We might have talked about this, but checking for a render is actually really difficult-- in this case, just checking for responding with success is good enough!